RELEASE_NOTES.md

```diff
@@ -1,5 +1,26 @@
-# Cosmos SDK v0.41.3 "Stargate" Release Notes
+# Cosmos SDK v0.41.4 "Stargate" Release Notes
 
-This release fixes a build failure caused by a small API breakage introduced in Tendermint v0.34.7. It is **highly recommended** that all applications using v0.41.2 upgrade to v0.41.3 as soon as possible.
+This release includes the addition of the multisign-batch command, minor bug fixes, and performance improvements.
+
+See the [Cosmos SDK v0.41.4 milestone](https://github.com/cosmos/cosmos-sdk/milestone/40?closed=1) on our issue tracker for details.
+
```

Comment:
quieter Tendermint logs deserves its own subsection, it's imo the most important change in this release

Reply:
You ok with this? https://github.com/cosmos/cosmos-sdk/pull/8748/files#diff-68d24fa81558aae3d8c59e2aa57a4fa719ea3b04d7fa14beff45f16f00858f50R19